src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -283,30 +286,20 @@ class Manager : public DoutPrefixProvider {
                         << " retry_number: "
                         << entry_persistency_tracker.retires_num
                         << " current time: " << time_now << dendl;
-    try {
-      // TODO move endpoint creation to queue level
-      const auto push_endpoint = RGWPubSubEndpoint::create(event_entry.push_endpoint, event_entry.arn_topic,
```

Comment:
eventually we shoud remove these fields from the event entry.
but for mixed cluster compatibility, we should just keep them

Reply:
> eventually we shoud remove these fields from the event entry. but for mixed cluster compatibility, we should just keep them

exactly, thats what we discussed, we keep them for now for compatibility and also populate them for now and we use them if for some reason we are not able to load the topic